requirements.txt

```diff
@@ -36,7 +36,7 @@ Werkzeug==1.0.1
 flask-cors
 pulp
 flask-apispec
```

Comment:
This change is so that it takes the change to the function in the PR in cornflow-cllient (https://github.com/baobabsoluciones/cornflow-client/pull/10), once that PR is accepted I will update this PR to point back to the cornflow-client master